### PR TITLE
PGBOUNCER_CONNECT_QUERY

### PR DESCRIPTION
**Pull request by anasinnyk**
**Description of the change**

This functionality add one more option in connection configuration `connect_query`

**Benefits**

We can change role after connection to database `set role "specific"`

**Additional information**

[more details](https://www.pgbouncer.org/config.html#connect_query) about this option in pgbouncer

### CODE REVIEW

**Review by moskitone — APPROVED** [review]

File: 1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh [hunk 1]
```diff
@@ -213,6 +213,9 @@ pgbouncer_initialize() {
         if ! is_empty_value "$PGBOUNCER_AUTH_USER"; then
             database_value+=" auth_user=$PGBOUNCER_AUTH_USER"
         fi
+        if ! is_empty_value "$PGBOUNCER_SET_CONNECT_QUERY"; then
```
Comment (joancafom) [inline 1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh:216]: Using `PGBOUNCER_SET_CONNECT_QUERY` as the name for the env var gives the idea of a boolean, just like `PGBOUNCER_SET_DATABASE_USER` (which allowed values are `yes`/`no`).

I rather use `PGBOUNCER_CONNECT_QUERY`:
```suggestion
        if ! is_empty_value "$PGBOUNCER_CONNECT_QUERY"; then
```

File: 1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh [hunk 2]
```diff
@@ -213,6 +213,9 @@ pgbouncer_initialize() {
         if ! is_empty_value "$PGBOUNCER_AUTH_USER"; then
             database_value+=" auth_user=$PGBOUNCER_AUTH_USER"
         fi
+        if ! is_empty_value "$PGBOUNCER_CONNECT_QUERY"; then
+            database_value+=" connect_query='$PGBOUNCER_CONNECT_QUERY'"
```
Comment (joancafom) [inline 1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh:217]: Use curly braces when concatenating env. vars with a regular string

```suggestion
            database_value+=" connect_query='${PGBOUNCER_CONNECT_QUERY}'"
```

**Review by joancafom — COMMENTED** [review]
Added a minor comment, but the overall PR looks good to me! Could you please apply this last feedback?

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for your PR 🚀

**Pull request merged**